### ISSUE TITLE
Question about training setup

### ISSUE THREAD

**Issue by entn-at** [usimarit/TiramisuASR#27]
Hi,

I'm curious to learn what training setup you are using to train the subword conformer librispeech model. I'm currently training such a model (on a different dataset) on a RTX2080Ti but noticed very low GPU utilization and generally slow training speed. I am trying to figure out how to improve training speed.

Thanks!

**Comment by usimarit** [comment]
I used 4 GPUs RTX 2080Ti, I trained whole LibriSpeech in 2 days using conformer with 1031 subwords (no tfrecords)
If you use HDD, you can speed up training by using tfrecords.
I noticed the GPU util is about 30-60% in my case.
I think it's because of the LSTMs and RNNT Loss that cause GPU utilization lower than models using only CNN.

**Comment by entn-at** [comment]
Thanks a lot!
I tried training with 4xRTX2080Ti, `config.yml` copied from the pretrained librispeech subword model (except minor changes to paths etc.) with 764 subwords and either with or without tfrecords (data is on SSD). Unfortunately, GPU utilization is practically zero most of the time, sometimes 1-3%. Consequently, the time per batch is quite high:
`[Train] [Epoch 1/20] |                    | 76/1125000 [1:22:11<16213:03:56, 51.89s/batch, transducer_loss=744.61957]`

It could be that the speech segments in my data are longer compared to librispeech. I'll run it with librispeech just to make sure that the rest of my setup is working as it should.

Thanks again!

**Comment by entn-at** [comment]
Just to follow up on this: Training is working fine now, I missed an outlier with long duration in my dataset. After removing it, training is progressing with ~2 batch/s.

Thanks again for the prompt reply!
